indy_node/server/domain_req_handler.py

```diff
@@ -210,14 +213,52 @@ def _validateAttrib(self, req: Request):
                                        'attribute for it'.
                                        format(TARGET_NYM))
 
-        if op.get(TARGET_NYM) and op[TARGET_NYM] != req.identifier and \
-                not self.idrCache.getOwnerFor(op[TARGET_NYM],
-                                              isCommitted=False) == origin:
-            raise UnauthorizedClientRequest(
-                req.identifier,
-                req.reqId,
-                "Only identity owner/guardian can add attribute "
-                "for that identity")
+        is_owner = self.idrCache.getOwnerFor(op[TARGET_NYM],
+                                             isCommitted=False) == origin
+        field = None
+        value = None
+        for key in (RAW, ENC, HASH):
+            if key in op:
+                field = key
+                value = op[key]
+                break
+        if field is None or value is None:
+            raise LogicError('Attribute data cannot be empty')
+
+        get_key = None
+        if field == RAW:
+            try:
+                get_key = json.loads(value)
```

Comment:
It will be better to use `attrib_raw_data_serializer` instead of `json`.

Reply:
ok